TestingLowerBounds/ForMathlib/Tendsto.lean

```diff
@@ -0,0 +1,23 @@
+import Mathlib.Topology.ContinuousOn
+
+open Filter Set
+
+open scoped  Topology
+
+variable {α β : Type*} [TopologicalSpace β] {f : α → β} {b : β} {l : Filter α}
+
+--I'm not sure if something that easily implies those lemmas already exists in mathlib, they can probably be proved in a simpler and more elegant way than I did.
+
+lemma tendsto_nhdsWithin_of_tendsto_nhds_right {t : Set β} (hf : ∀ᶠ x in l, f x ∈ t)
```

Comment:
[tendsto_nhdsWithin_of_tendsto_nhds_of_eventually_within](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/ContinuousOn.html#tendsto_nhdsWithin_of_tendsto_nhds_of_eventually_within)

Reply:
Thank you, I guess then this file is not needed anymore, I will delete it.